x-pack/plugin/sql/connectors/tableau/connector/connectionProperties.js

```diff
@@ -20,5 +20,7 @@
         }
     }
 
+    props["catalog"] = attr[connectionHelper.attributeDatabase]
```

Comment:
```suggestion
    props["catalog"] = attr[connectionHelper.attributeDatabase];
```

Probably doesn't matter but you never know...